src/cart/crt_iv.c

```diff
@@ -1070,21 +1099,31 @@ handle_ivfetch_response(const struct crt_cb_info *cb_info)
 	ivns = iv_info->ifc_ivns_internal;
 	class_id = iv_info->ifc_class_id;
 
+	/* Check to see if version change on while waiting for reponse */
+	/* Not an error. But good to know.			     */
+	local_version = ivns->cii_grp_priv->gp_membs_ver;
+	if (local_version != rpc_version) {
+		D_DEBUG(DB_ALL, "Group Version Changed: ");
+		D_DEBUG(DB_ALL, "rpc_version 0x%x: Local version 0x%x\n",
+				rpc_version, local_version);
+	}
+
 	iv_ops = crt_iv_ops_get(ivns, class_id);
 	D_ASSERT(iv_ops != NULL);
 
 	IV_DBG(&input->ifi_key, "response received, rc = %d\n", rc);
 
 	/* In case of a failure, call on_refresh with NULL iv_value */
 	iv_ops->ivo_on_refresh(ivns, &input->ifi_key,
-				0, /* TODO: iv_ver */
+				rpc_version,
```

Comment:
version here should be 0, this is an iv_version, which is currently not passed, and this is different from group versions